src/main/java/com/continuum/nova/input/Keyboard.java

```diff
@@ -192,96 +190,80 @@ public static void poll() {
         return keyName[key];
     }
 
     public static synchronized int getKeyIndex(String keyName) {
-        Integer ret = (Integer) keyMap.get(keyName);
-        return ret == null ? 0 : ret.intValue();
+        Integer ret = keyMap.get(keyName);
+        return ret == null ? 0 : ret;
     }
 
 
     public static boolean next() {
         key_press_event p = NovaNative.INSTANCE.get_next_key_press_event();
         key_char_event c = NovaNative.INSTANCE.get_next_key_char_event();
 
-        if (p.filled==0 && c.filled==0){
+        if (p.filled == 0 && c.filled == 0) {
             return false;
         }
-        if (p.filled==1 && p.key !=0 &&p.action!=2){
-            if (p.action ==1){
+        if (p.filled == 1 && p.key != 0 && p.action != 2) {
+            if (p.action == 1) {
                 keyDownBuffer.add(p.key);
 
-            } else{
+            } else {
                 keyDownBuffer.remove(p.key);
             }
         }
-        current_event.key = p.key;
-        current_event.character = (char) c.unicode_char;
-        current_event.state = p.action!=0;
-        current_event.repeat = p.action == 2;
+        currentEvent.key = p.key;
+        currentEvent.character = (char) c.unicode_char;
+        currentEvent.state = p.action != 0;
+        currentEvent.repeat = p.action == 2;
+
         return true;
     }
 
 
     public static void enableRepeatEvents(boolean enable) {
-
         repeat_enabled = enable;
-
     }
 
     public static char getEventCharacter() {
-
-        return  current_event.character;
-
+        return currentEvent.character;
     }
 
     public static int getEventKey() {
-
-        return current_event.key;
-
+        return currentEvent.key;
     }
 
     public static boolean getEventKeyState() {
-
-        return current_event.state;
-
+        return currentEvent.state;
     }
 
     public static boolean isRepeatEvent() {
-
-        return current_event.repeat;
-
+        return currentEvent.repeat;
     }
 
     static {
-        Field[] fields = Keyboard.class.getFields();
-
         try {
-            Field[] e = fields;
-            int len$ = fields.length;
-
-            for (int i$ = 0; i$ < len$; ++i$) {
-                Field field = e[i$];
-                if (Modifier.isStatic(field.getModifiers()) && Modifier.isPublic(field.getModifiers()) && Modifier.isFinal(field.getModifiers()) && field.getType().equals(Integer.TYPE) && field.getName().startsWith("KEY_") && !field.getName().endsWith("WIN")) {
-                    int key = field.getInt((Object) null);
+            for (Field field : Keyboard.class.getFields()) {
+                if (Modifier.isStatic(field.getModifiers()) && Modifier.isPublic(field.getModifiers()) &&
+                        Modifier.isFinal(field.getModifiers()) && field.getType().equals(Integer.TYPE) &&
+                        field.getName().startsWith("KEY_") && !field.getName().endsWith("WIN")) {
```

Comment:
This if statement is long and hard to read. Why not break it up so each condition is a different variable with a human-readable name?